packages/atomic/src/global/environment.ts

```diff
@@ -0,0 +1,13 @@
+import {VERSION} from '@coveo/headless';
+
+export interface AtomicEnvironment {
+  version: string;
+  headlessVersion: string;
+}
+
+export default function () {
+  ((window as unknown) as {atomic: AtomicEnvironment}).atomic = {
+    version: process.env.VERSION!,
+    headlessVersion: VERSION,
+  };
```

Comment:
I chose to expose the atomic version and headless version in the window since that was especially convenient in JSUI.

An alternative I tested was to expose a prop in search interfaces, giving the following result when a page is rendered:
```html
<atomic-search-interface id="search" version="0.8.0-alpha.23" headless-version="0.8.0-alpha.20" pipeline="default" search-hub="default" language="en" class="hydrated">
...
</atomic-search-interface>
```
However, I don't like the prop approach as much since it gives the false impression that the version can be chosen by changing the prop.

Reply:
Besides that, I have a question: do you prefer `window.atomic` or `window.coveo`?

Reply:
I prefer that window approach, besides, it might be used further in the future to manage multiple search interfaces using the same engines
`window.atomic` is a bit confusing with `window.Atomics` which is already a thing 🤔

Reply:
When loading Atomic as a UMD via a script tag, I imagine the script adds a global variable?

1. Is this the case?
2. If so, are we using the same global variable created by Stencil? or a different one? I imagine the name of the global can be configured too,

Reply:
1. Yes, it loads the script which add the global variable to the window, with all output types not just UMD
2. Should be our own, Ionic for example adds their own Ionic variable to the window

Reply:
Renaming it to `window.Coveo`.